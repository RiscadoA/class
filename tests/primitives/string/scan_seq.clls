proc main() {
    cut {
        send ch(x. scan(x));
        send ch(y. scan(y));
        send ch(z. scan(z));
        close ch
        |ch: ~send lstring; send lstring; send lstring; close|
        recv ch(x); println(x);
        recv ch(y); println(y);
        recv ch(z); println(z);
        wait ch; ()
    }
};;
