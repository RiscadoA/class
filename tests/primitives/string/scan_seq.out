a non-empty line

line above was empty
